packages/ra-core/src/util/TestContext.js

```diff
@@ -16,6 +16,12 @@ const defaultStore = {
     i18n: { locale: 'en', messages: {} },
 };
 
+const defaultReducers = {
```

Comment:
why don't you use react-admin's `createAdminStore` instead?

Reply:
ah nice - didn't realise that was there.  I'll check out `createAdminStore ` and update the PR in the morning.